x/launch/keeper/msg_server_create_chain.go

```diff
@@ -19,6 +19,8 @@ func (k msgServer) CreateChain(goCtx context.Context, msg *types.MsgCreateChain)
 		return nil, sdkerrors.Wrap(profiletypes.ErrCoordAddressNotFound, msg.Coordinator)
 	}
 
+	// TODO check metadata len
```

Comment:
I think we can do this now :)

Also since the check is static for now with an hardcoded value, the check can be performed in the static message validity check in `types` for better performance
[EDIT]: it is actually implemented

Reply:
Yeah I only left this comment if we want to instead use an on-chain param to store this max length.  But its not in the scope of this PR